libs/s25main/ingameWindows/iwBuildings.cpp

```diff
@@ -91,7 +95,7 @@ iwBuildings::iwBuildings(GameWorldView& gwv, GameCommandFactory& gcFactory)
 
     // "Help" button
     Extent btSize = Extent(30, 32);
-    AddImageButton(35, GetFullSize() - DrawPoint(14, 20) - btSize, btSize, TextureColor::Grey,
+    AddImageButton(38, GetFullSize() - DrawPoint(14, 20) - btSize, btSize, TextureColor::Grey,
```

Comment:
can you add a named constant for this (and possibly other) IDs? I.e. a file-scoped anonymous enum.
It might also be better to put all the building buttons last (with an offset) such that we don't need to "guess" the ID here.

Reply:
Done